conbench/tests/populate_local_conbench.py

```diff
@@ -398,14 +417,155 @@ def create_benchmarks_with_history():
                         mean=mean,
                     )
 
-                benchmark_id = post_benchmarks(benchmark_data)
+                benchmark_id = post_benchmark_result(benchmark_data)
                 benchmark_ids.append(benchmark_id)
                 runs.append((run_id, timestamp))
 
+
+def generate_synthetic_benchmark_history():
+
+    commits = [c.strip() for c in reversed(ARROW_COMMIT_HASH_LINES_50.splitlines())]
+    benchmark_name = "dummy-bench"
+
+    distr_mean = 20.0
+    # This is to simulate the real-world effect of there being a theoretical
+    # optimum: the fastest benchmark duration time with all sources of noise
+    # being silenced.
+    lower_bound = 17.5
+    slowdown_offset = distr_mean * 0.1
```

Comment:
sure, that's a great starting point for simulating a distribution change (picking a certain time window that is transformed by a certain shift, instead of picking an individual sample)